pylgbst/messages.py

```diff
@@ -601,11 +601,17 @@ def decode(cls, data):
     def _value(self):
         info = MsgPortModeInfoRequest
         if self.info_type == info.INFO_NAME:
-            return self.payload[:self.payload.index(b"\00")].decode('ascii')
+            try:
```

Comment:
Can you elaborate on why is this needed?

Reply:
Sorry, it's a leftover I forgot to remove. Good catch. I added these lines of code so I could run `describe_possible_modes` when analyzing the device. It turns out that the `decode('ascii')` function would crash at certain points in the decoding process. But that is not needed for the working code itself.